x-pack/plugins/maps/public/classes/sources/kibana_regionmap_source/kibana_regionmap_source.ts

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
There are some anys that need to be cleaned up in this file. Would it be better to just leave this file as js since its getting removed in 8.0?